frontend/src/routes/Search.tsx

```diff
@@ -585,7 +599,145 @@ const SearchSeries: React.FC<SearchSeriesProps> = ({ id, title, description, hos
+        readonly isLive: boolean;
+        readonly thumbnail: string | null | undefined;
+    };
+    title: string;
+}
+
+// This is using much of the same styles and logic as the `<Thumbnail>` component for single videos.
+// However, I think the differences are large enough to make modifying the original component to be
+// usable for both cases too much of a hassle. We could consider pulling out some shared css styles,
+// but again, there are subtle differences in most cases.
+const SeriesThumbnail: React.FC<SeriesThumbnailProps> = ({ info, title }) => {
+    const { t } = useTranslation();
+    const isDark = useColorScheme().scheme === "dark";
+
+    let inner;
+    if (info.thumbnail != null) {
+        // We have a proper thumbnail.
+        inner = <ThumbnailImg
+            src={info.thumbnail}
+            alt={t("series.entry-of-series-thumbnail", { series: title })}
+        />;
+    } else {
+        // We have no thumbnail. If the resolution is `null` as well, we are
+        // dealing with an audio-only event and show an appropriate icon.
+        // Otherwise we use a generic icon.
+        const icon = info.audioOnly ? <LuVolume2 /> : <LuFilm />;
+
+        inner = (
+            <div css={{
+                display: "flex",
+                height: "100%",
+                alignItems: "center",
+                justifyContent: "center",
+                fontSize: 40,
+                background: "linear-gradient(135deg, #33333380 50%, transparent 0),"
+                    + "linear-gradient(-135deg, #33333380 50%, transparent 0)",
+                backgroundSize: "17px 17px",
+                color: "#dbdbdb",
+                backgroundColor: "#292929",
+                ...isDark && {
+                    backgroundColor: "#313131",
+                    background: "linear-gradient(135deg, #3e3e3e80 50%, transparent 0),"
+                        + "linear-gradient(-135deg, #3e3e3e80 50%, transparent 0)",
+                },
+            }}>{icon}</div>
+        );
+    }
+
+    const overlay = <div css={{
+        display: "inline-flex",
+        alignItems: "center",
+        gap: 6,
+        position: "absolute",
+        right: 6,
+        bottom: 6,
+        borderRadius: 4,
+        padding: "1px 5px",
+        fontSize: 14,
+        backgroundColor: "rgba(200, 0, 0, 0.9)",
+        color: "white",
+    }}>
+        <LuRadio css={{ fontSize: 19, strokeWidth: 1.4 }} />
+        {t("video.live")}
+    </div>;
+
+    return (
+        <div css={{
+            position: "relative",
+            transition: "0.2s box-shadow",
+            overflow: "hidden",
+            height: "fit-content",
+            borderRadius: 8,
+            aspectRatio: 16 / 9,
+            ...isDark && {
+                img: {
+                    filter: "brightness(90%)",
+                    transition: "0.1s filter",
+                },
+            },
+        }}>
+            {inner}
+            {info.isLive && overlay}
+        </div>
```

Comment:
And maybe extracting this into `ThumbnailOverlayContainer` with just `childrens` prop?

Reply:
Thank you for these suggestions. I wonder whether the thumbnail stuff should live in its own file now that there are like 5 different thumbnail specific components that are used in both `Video.tsx` and `Search.tsx`. Though that would leave the `Video.tsx` file pretty empty.